docs/api-reference/full-text/index.mdx

```diff
@@ -157,7 +157,8 @@ CALL paradedb.create_bm25(
 ### JSON Fields
 
 Columns of type `JSON` and `JSONB` can be indexed as `json_fields`. Once indexed, search can be
-performed on nested text fields within JSON values.
+performed on nested text fields within JSON values. Note that the `JSON` and `JSONB` columns are
```

Comment:
What does this mean?

Reply:
There is not strict schema in the JSONB column that each row inserting some JSONB needs to conform to. I had a user mention they were unaware of this.

Reply:
Presumably if a user were indexing a JSON field they would already have created a JSON column and would know that? This section is about the BM25 index, not Postgres types.